geonode/layers/management/commands/update_styles_with_empty_title.py

```diff
@@ -0,0 +1,29 @@
+from os import name
+from geonode.layers.models import Style
+from django.core.management.base import BaseCommand
+
+from argparse import RawTextHelpFormatter
+
+from geonode.layers.models import Style
+from geonode.geoserver.helpers import gs_catalog
+
+
+class Command(BaseCommand):
```

Comment:
We don't need to incorporate this special command in the repo. We can execute the steps on the target intance. Please report the commands inside the original issue https://github.com/geosolutions-it/unesco-ihp/issues/353

Reply:
Done